package.json

```diff
@@ -18,19 +18,18 @@
         "migrate": "yarn knex migrate:latest && yarn typeorm migration:run",
         "lint": "eslint .",
         "fix": "eslint . --fix",
-        "prettify": "yarn pretty-quick --pattern \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss}\"",
-        "prettify:check": "yarn pretty-quick --check --pattern \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss}\"",
-        "prettify-all": "yarn prettier --write \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss}\"",
-        "prettify-all:check": "yarn prettier --check \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss}\"",
+        "prettify": "yarn pretty-quick --pattern \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss,yml}\"",
+        "prettify:check": "yarn pretty-quick --check --pattern \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss,yml}\"",
```

Comment:
Nit: Should we just ditch `yarn prettify:check` and `yarn prettify-all:check` since they are just dupes? I think `yarn prettify --check` and `yarn prettify-all --check` would accomplish the same.

Reply:
good point! I think it still makes sense to keep the `:check` variants around for documentation purposes. The  `prettify:check` could be done with less repetition though, so it's now simplified, thanks!